experimental/dds/tree2/src/shared-tree/sharedTreeView.ts

```diff
@@ -67,11 +67,13 @@ export interface ViewEvents {
 	 * The event along with the {@link LocalCommitSource} communicates a change has been made that can be undone or redone on
 	 * the {@link ISharedTreeView}. However, the {@link ISharedTreeView} completely manages its own undo/redo
 	 * stack which cannot be modified and no additional information about the change is provided.
-	 *
-	 * Merging and rebasing branches can cause the order of revertibles in the undo/redo stack to change.
-	 * Therefore, the order of revertible events are not guaranteed to be reflective of the order of the undo/redo
-	 * stack in these cases. The only exception is merging a branch into the local branch because the order of commits is
-	 * not changed in this case.
+	 * 
+	 * Revertible events are emitted when merging a view into another view but not when rebasing onto another view. This is because
```

Comment:
```suggestion
	 * Revertible events are emitted when merging a view into this view but not when rebasing this view onto another view. This is because
```